### PR TITLE
Small Tweaks to fix warning, update description text, etc.

### PR DESCRIPTION
**Pull request by philcrump**
- ubxtool.cc: Change logic order of exception handling in ubx functions to remove compiler warning (no change in behaviour)
 - ubxtool.cc: Hide SBAS "Preamble" stdout message behind debug flag
 - .gitignore: Add navparse binary.
- html/: Reword "four receivers hosted at <incomplete list of locations>" to "receivers hosted in locations around the globe."
- navnexus.cc,navdisplay.cc: Explicitly cast time()-derived values to uint64_t to suppress compile warnings. time() will return -1 and hence overflow if there's an error with time(), however this is unlikely and the overflow itself is not fatal to the application.
 - html/: Add HTML titles and description meta fields.
 - html/overview.html,overview.js: Remove unused and broken overview.html page and relevant js.

### CODE REVIEW

File: ubxtool.cc [hunk 1]
```diff
@@ -230,12 +230,12 @@ UBXMessage sendAndWaitForUBX(int fd, int seconds, basic_string_view<uint8_t> msg
...
-      if(n==1)
-        throw;
-      cerr<<"Retransmit"<<endl;
-          
+      if(n > 1) {
+        cerr<<"Retransmit"<<endl;
+      }
     }
   }
+  throw;
```
Comment (ahupowerdns) [inline ubxtool.cc]: I don't understand this - what does this 'throw' do? Does this even compile? I may be dense or not reading it right but it looks weird.

Comment (philcrump) [reply ubxtool.cc]: I moved it from line 234. It's what you're using to throw an exception when 3 attempts time-out. Moving it here to the end of the function however allows the compiler to see that the function either returns the ubxmessage, or throws, which silences the compile warning. The behaviour of the function is unchanged as the for() loop exits after a third unsuccessful attempt.

Comment (philcrump) [reply ubxtool.cc]: My misunderstanding of catch(...) { throw; } behaviour in propagating the caught exception. Commit reverted.

**Comment by philcrump** [comment]
Several of these changes have now been made by others, hence merge conflicts. Closing and will open new individual pull reqs for anything notably outstanding.

**Pull request closed without merging**